docker/pypi/wmagent-mariadb/mariadb-docker-run.sh

```diff
@@ -69,6 +69,7 @@ dockerOpts="
 --detach \
 --network=host \
 --rm \
+--user $UID:$GID \
```

Comment:
@vkuznet  just wanted to mention, since we are using bash as  default shell in our agents, the `$GUID` env variable is empty. The best possible way to get the group id would be `id -g`.

Reply:
Todor, I applied your suggestion.